tests/MyGame/Example/Monster.go

```diff
@@ -99,6 +99,15 @@ func (rcv *Monster) InventoryBytes() []byte {
 	return nil
 }
 
+func (rcv *Monster) MutateInventory(j int, n byte) bool {
```

Comment:
The one sad part of this is that is will generate a lot of overhead if you loop through a vector, since it obtains the vector every time. But with the current API there is no alternative I guess, and it is better to have the option than not.

Reply:
It would be possible to add a function like `MutateInventoryBySlice(offset int, values []byte) bool` that sets a sub-vector to given slice with less overhead. Actually I am using it in a loop in my application so I would use that version if it existed, though I'm dealing with small vectors so it doesn't make much difference.

Reply:
@dnr @aardappel maybe we can do that in another PR?